unit-tests/live/safety/test-safety-preset-read-write-real-data.py

```diff
@@ -0,0 +1,98 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun
+#test:device D585S
+
+import pyrealsense2 as rs
+import random
+from rspy import test
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Verify Safety Sensor Extension")
+ctx = rs.context()
+dev = ctx.query_devices()[0]
+safety_sensor = dev.first_safety_sensor()
+test.finish()
+
+#############################################################################################
+
+test.start("New safety preset creation")
```

Comment:
This is not a test, its a preparation of data.
Better add this as a function
`get_random_preset' and call it inside the test

Reply:
done